package.json

```diff
@@ -21,6 +21,7 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "publish": "lerna run build && lerna publish"
+    "publish": "lerna run build && lerna publish",
+    "test": "lerna exec npx jest -- --coverage"
```

Comment:
**Discussion:**

What are the pro/cons of using `lerna exec npx jest` vs `lerna run test` and having a `test` script inside of each package?

That way if a package needs to run a different sort of test command they could.